VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx

```diff
@@ -287,8 +288,10 @@ const PrescriptionHistory: FC<PrescriptionHistoryProps> = ({ navigation, route }
   }
 
   const prescriptionDetailsClicked = (prescriptionID: string) => {
-    logAnalyticsEvent(Events.vama_rx_details(prescriptionID))
-    return navigation.navigate('PrescriptionDetails', { prescriptionId: prescriptionID })
+    if (waygateNativeAlert('WG_PrescriptionDetails')) {
+      logAnalyticsEvent(Events.vama_rx_details(prescriptionID))
```

Comment:
I confirmed this with Greta because I made a call and I wanted to make sure it was the right one, but in these situations with previously existing logAnalytics we should log the analytic BEFORE the waygate check, because even if we stop the nav we want to log that they tried.

Reply:
That makes sense. I went ahead and updated the appropriate screens to include that